DevelopGitHub/GitHubExtension/GitHubExtension.Security.WebApi.Library/Controllers/RepositoryController.cs

```diff
@@ -22,9 +23,9 @@ public RepositoryController(IGithubService guGithubService, ISecurityContext sec
             _securityContext = securityContext;
```

Comment:
ISecyrityCoOntext shold me removed from injecting in controller. Please cooperate with your colleauges regarding that.